documentation/modules/deploying/con-deploy-kafka-connect-managing-connectors.adoc

```diff
@@ -19,37 +19,39 @@ Sink connector:: A sink connector is a runtime entity that fetches messages from
 
 Strimzi provides two APIs for creating and managing connectors:
 
-* `KafkaConnector` resources (referred to as `KafkaConnectors`)
+* KafkaConnector resources (referred to as KafkaConnectors)
 * Kafka Connect REST API
 
 Using the APIs, you can:
 
 * Check the status of a connector instance
 * Reconfigure a running connector
 * Increase or decrease the number of tasks for a connector instance
-* Restart failed tasks (not supported by `KafkaConnector` resource)
+* Restart failed tasks (not supported by KafkaConnector resource)
 * Pause a connector instance
 * Resume a previously paused connector instance
 * Delete a connector instance
 
-== `KafkaConnector` resources
+== KafkaConnector resources
 
-`KafkaConnectors` allow you to create and manage connector instances for Kafka Connect in a Kubernetes-native way, so an HTTP client such as cURL is not required.
-Like other Kafka resources, you declare a connector’s desired state in a `KafkaConnector` YAML file that is deployed to your Kubernetes cluster to create the connector instance.
+KafkaConnectors allow you to create and manage connector instances for Kafka Connect in a Kubernetes-native way, so an HTTP client such as cURL is not required.
+Like other Kafka resources, you declare a connector’s desired state in a KafkaConnector YAML file that is deployed to your Kubernetes cluster to create the connector instance.
```

Comment:
I would use the monospace here because we are referring to a YAML file.

Reply:
But not the name of the YAML file? A YAML file for KafkaConnector.